### DISCUSSION TITLE
"Stake & Delegate" and "Plan & Offer" the transaction history hard to understand

### DISCUSSION THREAD

**Discussion by zhephirothz** [subquery/subql#1018] [Frontier TestNet - Bug Reporting]
On the page "Stake & Delegate"
There is no page to summarize my status which one I do stake or which one stake on me.

Same as the page "Plan & Offer"
I don't know which wallet purchase my plan or which plan I purchased.
It's show all and I don't know which one is my sales or my purchase.

**Comment by mzxyz** [comment]
@lilian79511
